gradle/scripts/lib/java.gradle

```diff
@@ -1,11 +1,13 @@
-def buildJdkVersion = 17
+def buildJdkVersion = Integer.parseInt(JavaVersion.current().getMajorVersion())
```

Comment:
Note: I believe this is a slight change in behavior since the jvm version which gradle is run with will be used by default instead of 17.

I agree with this change though, since I dislike having the most recent jdk version fragmented across multiple files 🙇  Just wanted to note this change 👍